habushu-maven-plugin/src/main/resources/migrations.json

```
@@ -1,26 +1,41 @@
 [
-    {
-        "group": "baton-migrations", 
-        "type": "ordered", 
-        "migrations": [
-            {
-                "name": "remove-monorepo-group-migration",
-                "implementation": "org.technologybrewery.habushu.migration.RemoveMonorepoGroupMigration",
-                "fileSets": [
-                  {
-                    "includes": ["pyproject.toml"]
-                  }
-                ]
-              },
-              {
-                "name": "custom-poetry-core-version-migration",
-                "implementation": "org.technologybrewery.habushu.migration.CustomPoetrycoreVersionMigration",
-                "fileSets": [
-                  {
-                    "includes": ["pyproject.toml"]
-                  }
-                ]
-              }
+  {
+    "group": "baton-migrations",
+    "type": "ordered",
+    "migrations": [
+      {
+        "name": "remove-monorepo-group-migration",
+        "implementation": "org.technologybrewery.habushu.migration.RemoveMonorepoGroupMigration",
+        "fileSets": [
+          {
+            "includes": [
+              "pyproject.toml"
+            ]
+          }
         ]
-    }
+      },
+      {
+        "name": "custom-poetry-core-version-migration",
+        "implementation": "org.technologybrewery.habushu.migration.CustomPoetrycoreVersionMigration",
+        "fileSets": [
+          {
+            "includes": [
+              "pyproject.toml"
+            ]
+          }
+        ]
+      },
+      {
+        "name": "dev-dependencies-preferred-poetry-approach-group-migration",
```

A: I personally think this reads better, but no worries if we think the original naming is better 

```suggestion
        "name": "preferred-poetry-approach-dev-dependencies-group-migration",
```

---

TomAto-tOmato - gonna leave it the way it is